containers/ecr-viewer/src/instrumentation.ts

```diff
@@ -12,45 +14,43 @@ export async function register() {
 }
 
 function setupConfigurationVariables() {
+  const sourceMap: {
+    [key: string]: typeof S3_SOURCE | typeof AZURE_SOURCE | typeof GCP_SOURCE;
```

Comment:
out of curiosity does `typeof process.env.SOURCE` work here as that has already been defined with these sources (I think maybe)